tests/acceptance/features/apiGraph/deleteUser.feature

```diff
@@ -13,7 +13,7 @@ Feature: delete user
 
   Scenario Outline: admin user deletes a user
     Given the administrator has assigned the role "Admin" to user "Alice" using the Graph API
-    And the user "Alice" has created a new user using the Graph API with the following settings:
+    And the administrator has created a new user with the following attributes:
```

Comment:
When this step says "the administrator" it is talking about the user "Alice" who has been given the role "admin" in the above step or is it about the "admin" user? I don't think it's user "Alice" here or is it?

Reply:
The scenario is about the admin user deleting another user. So, IMO it won't make any difference if the main administrator created the new user. Or Should I switch the line of these two steps to remove the confusion?